CIP-1855/README.md

```diff
@@ -1,36 +1,37 @@
 ---
 CIP: 1855
 Title: Forging policy keys for HD Wallets
-Authors: Samuel Leathers <samuel.leathers@iohk.io>, John Lotoski <john.lotoski@iohk.io>, Michael Bishop <michael.bishop@iohk.io>, David Arnold <david.arnold@iohk.io>
-Comments-Summary: Multi-party transaction signing and key management for HD wallets.
-Comments-URI: https://github.com/cardano-foundation/CIPs/wiki/Comments:CIP-1855
 Status: Active
-Type: Standards
+Category: Wallets
+Authors:
+  - Samuel Leathers <samuel.leathers@iohk.io>
+  - John Lotoski <john.lotoski@iohk.io>
+  - Michael Bishop <michael.bishop@iohk.io>
+  - David Arnold <david.arnold@iohk.io>
+Implementors: N/A
```

Comment:
Unsure about the extent of implementations

Reply:
Somehow we recorded in our CIP editor communications that "everybody implements it" (continuing in front-line discussion with https://github.com/cardano-foundation/CIPs/pull/729#issuecomment-1873538406)

Reply:
It'd rather be [] here IMO, as this is something implementable but we are simply not aware of any implementor.

Reply:
OK, set back to `Proposed` and implementors = the empty set. https://github.com/cardano-foundation/CIPs/pull/729#issuecomment-1878045424